DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/matmul/src/CMakeLists.txt

```diff
@@ -0,0 +1,181 @@
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+###############################################################################
+### Customize these build variables
+###############################################################################
+set(SOURCE_FILE matmul_demo.cpp)
+set(TARGET_NAME matmul)
+
+set(ROWS_A 8)
```

Comment:
These need to be potentially set when calling cmake
e.g. `cmake .. -DSET_ROWS_A=16`
The cmake code would look like:
```
set(ROWS_A 8)
if(DEFINED SET_ROWS_A)
    set(ROWS_A ${SET_ROWS_A})
endif()
```
Of course you need to do this for the following variables as well + update your README